verticapy/jupyter/extensions/sql_magic.py

```diff
@@ -57,19 +57,606 @@ def sql_magic(
+        GROUP BY 1;
+
+    **Execution**: 0.006s
+
+    .. ipython:: python
+        :suppress:
+
+        import verticapy as vp
+        from verticapy.datasets import load_titanic, load_iris
+        titanic = load_titanic()
+        iris = load_iris()
+        %load_ext verticapy.sql
+
+    .. ipython:: python
+        :suppress:
+
+        %sql -c 'SELECT survived, AVG(fare) AS avg_fare, AVG(age) AS avg_age FROM titanic GROUP BY 1;'
+
+    .. ipython:: python
+        :suppress:
+
+        t = _
+        html_file = open("figures/jupyter_extensions_sql_magic_sql_magic.html", "w")
+        html_file.write(t._repr_html_())
+        html_file.close()
+
+    .. raw:: html
+        :file: SPHINX_DIRECTORY/figures/jupyter_extensions_sql_magic_sql_magic.html
+
+    You can also run queries with '%sql' and the '-c' option:
+
+    .. code-block:: python
+
+        %sql -c 'SELECT DISTINCT Species FROM iris;'
+
+    **Execution**: 0.006s
+
+    .. ipython:: python
+        :suppress:
+
+        %sql -c 'SELECT DISTINCT Species FROM iris;'
+
+    .. ipython:: python
+        :suppress:
+
+        result = _
+        html_file = open("figures/jupyter_extensions_sql_magic_sql_magic_2.html", "w")
+        html_file.write(result._repr_html_())
+        html_file.close()
+
+    .. raw:: html
+        :file: SPHINX_DIRECTORY/figures/jupyter_extensions_sql_magic_sql_magic_2.html
+
+    You can use a single cell for multiple queries:
+
+    .. code-block:: python
+
+        %%sql
+        DROP TABLE IF EXISTS test;
+        CREATE TABLE test AS SELECT 'Badr Ouali' AS name;
+        SELECT * FROM test;
+
+    **Execution**: 0.05s
+
+    .. ipython:: python
+        :suppress:
+        :okwarning:
+
+        %%sql
+        DROP TABLE IF EXISTS test;
+        CREATE TABLE test AS SELECT 'Badr Ouali' AS name;
+        SELECT * FROM test;
+
+    .. ipython:: python
+        :suppress:
+
+        test_table = _
+        html_file = open("figures/jupyter_extensions_sql_magic_sql_magic_3.html", "w")
+        html_file.write(test_table._repr_html_())
+        html_file.close()
+
+    .. raw:: html
+        :file: SPHINX_DIRECTORY/figures/jupyter_extensions_sql_magic_sql_magic_3.html
+
+    To add comments to a query, use one of the following comment syntaxes:
+
+    .. code-block:: python
+
+        %%sql
+        -- Comment Test
+        /* My Vertica Version */
+        SELECT version(); -- Select my current version
+
+    **Execution**: 0.005s
+
+    .. ipython:: python
+        :suppress:
+
+        %%sql
+        -- Comment Test
+        /* My Vertica Version */
+        SELECT version(); -- Select my current version
+
+    .. ipython:: python
+        :suppress:
+
+        test_comment = _
+        html_file = open("figures/jupyter_extensions_sql_magic_sql_magic_4.html", "w")
+        html_file.write(test_comment._repr_html_())
+        html_file.close()
+
+    .. raw:: html
+        :file: SPHINX_DIRECTORY/figures/jupyter_extensions_sql_magic_sql_magic_4.html
+
```

Comment:
.. warning:: Vertica uses '/*' and '*/' for both comments and query hints. Whenever possible, use '--' to avoid conflicts.